code/style.css

```diff
@@ -1,4 +1,157 @@
+body{
+  margin:0;
+}
 html {
-  background: #0025ff;
-  color: #fff;
+  color: black;
+  font-family: Helvetica, sans-serif, Arial,;
+  font-size: 12px;
+  background-color: white;
+  
+}
+
+.logo {
+  width: 100px;
+  height: 100px;
+  position: sticky;
+  border-radius: 20px ;
+  margin: 30px;
+  box-shadow: 10px 5px 5px teal;
+
+}
+
+header{
+  width: 100%;
+  height: 200px;
+  background-color: white;
+
+}
+nav{
+  display:flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 20px;
+}
+.site-name{
+  font-size: 6vw;
+  position: center;
+
+}
+li{
+  padding: 10px;
+  position: -webkit-sticky;
+  border-right: 20px;
+}
+h2:hover {
+  background-color: #254e58;
+}
+
+.big-news{
+  display: flex;
+  flex-direction: row;
+  margin: 10px 20px;
+  padding: 20px;
+  margin-right:10px;
+  text-align: center;
+
+  background-color: rgb(250, 250, 250);
+  align-items: left;
+}
+
+.big-news-image{
+  width: 100%;
+  
+}
+.text-container{
+  display: flex;
+  flex-direction:column;
+
+}
+
+.big-news-text{
```

Comment:
In some css styles when I click them out in the inspector it does not happen anything. An example here! 
Maybe good to go through the code and find code that is not neccessary. (I had a lot of that...) This example might be a bad one, since the img is a bit wide though. 
```
.big-news-text {
    text-align: center;
    flex-grow: 1;
}
```

Reply:
flex-direction:column;

or

flex-direction: column;

Reply:
See what I mean? :-)